sbnobj/Common/Reco/BarycenterMatch.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file   sbnobj/Common/Reco/BarycenterMatch.h
+ * @brief  Container for the result of matching reconstructed OpFlashes with Pandora charge slices via the barycenter method
+ * @author Jack Smedley (jsmedley@fnal.gov)
+ * @date   June 21, 2023
+ * 
+ */
+
+#include <limits>
+#include "larcoreobj/SimpleTypesAndConstants/geo_vectors.h"
+
+
+#ifndef BarycenterMatch_H
+#define BarycenterMatch_H
+
+namespace sbn {
+
+  class BarycenterMatch {
```

Comment:
I do have a brief already at the top of the file, lines 1-7. Is this not sufficient? Happy to update it as you see fit

Reply:
My experience with navigating the documentation is that I end clicking on the class names in the code rather than at the source file. Having to choose, I would stick the documentation to the class. This approach can also support library headers which have multiple classes or utilities.